tests/e2e/tools/php-errors.test.js

```diff
@@ -0,0 +1,75 @@
+const { test, expect } = require('@playwright/test');
+
+test('check admin PHP errors', async ({ page }) => {
+  test.setTimeout(240 * 1000);
+
+  await page.goto('/wp-admin/');
+
+  await page.locator('#user_login').focus();
+  await page.type('#user_login', 'admin');
+  await page.locator('#user_pass').focus();
+  await page.type('#user_pass', 'admin');
+  await page.getByText('Log In').click();
+
+  await expect(page.locator('#wpadminbar')).toBeVisible();
```

Comment:
I need to understand something, I thought the `login` function was to handle this process, why didn't you use it here?

Reply:
You're right, that's because i wrote that script before the login one :grimacing: